presto-main/src/main/resources/webapp/index.html

```diff
@@ -49,6 +49,7 @@
             <th>Source</th>
             <th>User</th>
             <th>State</th>
+            <th>Error Type</th>
```

Comment:
I tried this out and the header ends up getting wrapped into two lines when there aren't any failed queries, so it looks weird. Lets just call this "Error" to avoid that.